contracts/facets/EmbalmerFacet.sol

```diff
@@ -22,7 +22,7 @@ contract EmbalmerFacet {
         address[] cursedArchaeologists,
         uint256 totalDiggingFees,
         uint256 createSarcophagusProtocolFees,
-        string[] arweaveTxIds
+        string[2] arweaveTxIds
```

Comment:
Typing this as a tuple enforces the validation we were doing to be sure that 2 ids are passed in